features.js

```diff
@@ -759,10 +759,25 @@ const features = [
       },
     ],
   },
+  {
+    name: "Variadic functions / methods (... operator)",
+    description:
+      "Variadic functions can now be implemented using the <code>...</code> operator, instead of relying on `func_get_args()`.",
+    keywords: ["variadic", "argument", "parameter"],
+    added: "5.6",
+    deprecated: null,
+    removed: null,
+    resources: [
+      {
+        name: "Variadic functions via ...",
+        url: "https://www.php.net/manual/en/migration56.new-features.php#migration56.new-features.variadics",
+      },
+    ],
+  },
   {
     name: "Splat operator / argument unpacking (... operator)",
     description:
-      "The `...` operator, aka the splat operator or array unpacking operator",
+      "The <code>...</code> operator, aka the splat operator or array unpacking operator",
```

Comment:
I'll leave this in, but note that, [as per the README](https://github.com/rosswintle/can-i-php#adding-php-features), backticks are converted to `<code>` tags.

Is there some reason you change these? Was this not working?

Reply:
Oh, good to know. I noticed that both syntax were used and I naively thought that the backticks must be a mistake. Reverting this change.

Reply:
Our changes crossed. I think I've un-reverted it. 😆